http/client.go

```diff
@@ -0,0 +1,412 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package http
```

Comment:
question: The current (develop branch) `http` package has a playground.go file, I'm not familiar with the playground stuff at all, has this been forgotten about? Is playground.go deadcode? Have I just missed it somewhere in this PR?

Reply:
The follow up PR addresses this. 

https://github.com/sourcenetwork/defradb/blob/19c734bc7a6da1eba2e2ee108182a14bd41f5f39/http/handler_playground.go

Reply:
Cheers